spec/shared_examples_for_cinema_halls.rb

```diff
@@ -0,0 +1,13 @@
+# frozen_string_literal: true
+
+RSpec.shared_examples 'unauthorized' do |_parameter|
+  let(:role) { create(:user_role) }
+  let(:user) { create(:user, user_role: role) }
```

Comment:
What's the purpose of adding this lets? You are not using them as far as I see.

Reply:
Classic opy and paste mistake :D I have removed it, thank you!